deb/list.go

```diff
@@ -138,7 +138,7 @@ func (l *PackageList) Add(p *Package) error {
 	existing, ok := l.packages[key]
 	if ok {
 		if !existing.Equals(p) {
-			return &PackageConflictError{fmt.Errorf("conflict in package %s", p)}
+			return &PackageConflictError{fmt.Errorf("package already exists and is different: %s", p)}
```

Comment:
Should we say `is different` or `hash mismatch` ? 🤔 

Also, maybe we can add a info or debug log line to display the expected hash (and file size) vs the given one ?

Reply:
unfortunately there are many fields responsible for "different", logging each one seems overkill. 

see deb/package.go:
```
// Equals compares two packages to be identical
func (p *Package) Equals(p2 *Package) bool {
        return p.Name == p2.Name && p.Version == p2.Version && p.SourceArchitecture == p2.SourceArchitecture &&
                p.Architecture == p2.Architecture && p.Source == p2.Source && p.IsSource == p2.IsSource &&
                p.FilesHash == p2.FilesHash
}
```

the term `file already exists and is different` is already used in aptly, so at least it is more consistent...

would you extend this to log all the differing fields ?